lib/config/presets/npm/index.spec.ts

```diff
@@ -2,8 +2,6 @@ import * as httpMock from '../../../../test/http-mock';
 import { GlobalConfig } from '../../global';
 import * as npm from '.';
 
-jest.mock('delay');
```

Comment:
we need to mock the setTimeout or use jest fake timers, otherwise tests slows down unnecessarily

Reply:
Why was `delay` mocked in this test file when the corresponding ts file doesn't contain any timeouts? Neither do any imported functions in the file use timeout.